src/server/lib/queryParams.ts

```diff
@@ -3,12 +3,12 @@ import { validateReturnUrl, validateRefUrl } from '@/server/lib/validateUrl';
 import { validateClientId } from '@/server/lib/validateClientId';
 import { FederationErrors } from '@/shared/model/Errors';
 
-const validateEmailVerified = (emailVerified?: string): boolean | undefined => {
-  if (!emailVerified) {
+const validateBoolean = (maybeBoolean?: string): boolean | undefined => {
```

Comment:
could this one be something like `isStringTrue` maybe? had to read the function to work out what it did, although it was already quite ambiguous before the name change lol

Reply:
i've changed it to `isStringBoolean`!